### PR TITLE
Added the API getByIdAndStatus

### PR DESCRIPTION
**Pull request by ruksana2808**

### CODE REVIEW

File: src/main/java/com/igot/cb/designation/controller/DesignationController.java [hunk 1]
```diff
@@ -30,6 +36,19 @@ public ResponseEntity<String> loadJobsFromExcel(@RequestParam("file") MultipartF
...
 
+  @GetMapping("/read/{id}")
+  public ResponseEntity<?> playListRead(@PathVariable String id) {
+    CustomResponse response = designationService.readDesignation(id);
+    return new ResponseEntity<>(response, response.getResponseCode());
+  }
+
   //update API to store the refNodes
 
+  @PutMapping("/update/designation")
```
Comment (karthik-tarento) [inline src/main/java/com/igot/cb/designation/controller/DesignationController.java]: use the proper rest path... 

to the controller class -- it should be -> "/designation/v1"
to the methods -> "/update"

File: src/main/java/com/igot/cb/designation/service/DesignationService.java [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
 
+import com.fasterxml.jackson.databind.JsonNode;
+import com.igot.cb.pores.dto.CustomResponse;
+import com.igot.cb.pores.util.ApiResponse;
 import org.springframework.web.multipart.MultipartFile;
 
 public interface DesignationService {
 
- public void loadDesignationFromExcel(MultipartFile file);
+  public void loadDesignationFromExcel(MultipartFile file);
```
Comment (karthik-tarento) [inline src/main/java/com/igot/cb/designation/service/DesignationService.java]: We suppose to use csv ?

**Review by karthik-tarento — CHANGES_REQUESTED** [review]

File: src/main/java/com/igot/cb/designation/controller/DesignationController.java (hunk 1 above)
Comment (ruksana2808) [reply src/main/java/com/igot/cb/designation/controller/DesignationController.java]: made the changes

File: src/main/java/com/igot/cb/designation/service/DesignationService.java (hunk 2 above)
Comment (ruksana2808) [reply src/main/java/com/igot/cb/designation/service/DesignationService.java]: yeah we are uploading the csv now, implemented the csv upload in the method

**Review by sharathkashyap — APPROVED** [review]

**Pull request merged**